# Glass gaze
Header header
float32 last_step_x
float32 last_step_y
float32 last_step_z
float32 next_step_x
float32 next_step_y
float32 next_step_z